roles/fluentd_node/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+# TODO: Update fluentd install and configuration when packaging is complete
+- name: download and install td-agent
+  yum:
+    name: 'http://packages.treasuredata.com/2/redhat/7/x86_64/td-agent-2.2.0-0.x86_64.rpm'
+    state: present
+
+- name: install Kubernetes fluentd plugin
+  command: '/opt/td-agent/embedded/bin/gem install fluent-plugin-kubernetes'
+
+- name: Override td-agent configuration file
+  template:
+    src: td-agent.j2
+    dest: /etc/sysconfig/td-agent
+    group: 'td-agent'
+    owner: 'td-agent'
+    mode: 0444
+
+- name: Creates directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    group: 'td-agent'
+    owner: 'td-agent'
+    mode: 0755
+  with_items: ['/etc/td-agent/config.d', '/var/log/td-agent/tmp']
```

Comment:
We'll talk about this at the next meeting, but just a quick heads up. I'm going to propose that we stay in yaml as much as possible and only do json when we have to. Ansible outputs MUCH better error messages with yaml than with json.